app/views/layouts/exchanges_base.html.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :pages_layout do %>
```

Comment:
one note I'd make - you should be able to name this anything you want (and use the same name in the 'single_column' layout) - `pages_layout` made sense in the context of the example article (she was injecting static content from a 'pages' directory, IIRC), but maybe something like `child_layout`, `nested_layout`, `main_content`, or something makes more sense here?

Reply:
I liked `pages_layout`, but when you pointed out this I decided to name it `exchanges_layout`.
Thank you